lib/Agrammon/LanguageParser.pm6

```diff
@@ -0,0 +1,15 @@
+use v6;
+
+sub parse-lang-values(Str $value --> Hash) is export {
+    my %opt-lang;
+    for (split("\n", $value)) -> $ol {
+        my ($l, $o) = split(/ \s* '=' \s* /, $ol);
+        if not $o {
+            warn "ol=$ol";
```

Comment:
Maybe make it something like `"Failed to parse language value; ol=$ol"`, so it's easier to understand what the warning is about when it happens in 6 months time. :-)

Reply:
Indeed ... done.